move-mutation-test/src/lib.rs

```diff
@@ -116,35 +113,26 @@ pub fn run_mutation_test(
                 mutant_file.display()
             );
 
-            // Strip prefix to get the path relative to the package directory (or take that path if it's already relative).
-            let original_file = elem
-                .original_file_path()
-                .strip_prefix(&package_path)
-                .unwrap_or(elem.original_file_path());
-            let job_work_dir = format!("mutation_test_{rayon_thread_id}");
-            let outdir = outdir.join(job_work_dir);
+            // Strip prefix to get the path relative to the package directory.
+            let original_file =
+                strip_path_prefix(elem.original_file_path()).expect("invalid package path");
 
-            let _ = fs::remove_dir_all(&outdir);
-            move_mutator::compiler::copy_dir_all(&package_path, &outdir)
+            let job_outdir = outdir.join(format!("mutation_test_{rayon_thread_id}"));
+            let _ = fs::remove_dir_all(&job_outdir);
+
+            fs_extra::dir::copy(&package_path, &job_outdir, &cp_opts)
                 .expect("copying directory failed");
 
             trace!(
                 "Copying mutant file {} to the package directory {:?}",
                 mutant_file.display(),
-                outdir.join(original_file)
+                job_outdir.join(&original_file)
             );
-
             // Should never fail, since files will always exists.
-            let _ = fs::copy(mutant_file, outdir.join(original_file));
-
-            if let Err(e) =
-                move_mutator::compiler::rewrite_manifest_for_mutant(&package_path, &outdir)
```

Comment:
I think... I think we can do this only once after we copy the package path to the tempdir.

After we rewrite the manifest for that `original_package` cloned dir, every other mutant_dir will automatically contain manifest with absolute paths and everything should work fine